MDC-102 Web: Material Structure and Layout (Web) Too many issues in the codelab

This codelab tutorial has not been properly updated! Please updated the proper working and steps required to learn MDC. We are falling into many compile errors


 
ERROR in ./home.js
    Module not found: Error: Can't resolve '@material/list' in 'C:\Users\Desktop\mdc\material-components-web-codelabs-master\mdc-102\starter'       
     @ ./home.js 3:12-37
     @ multi (webpack)-dev-server/client?http://localhost:8080 ./home.js

--- a/mdc-101/complete/package.json
+++ b/mdc-101/complete/package.json
@@ -11,7 +11,7 @@
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-101/starter/package.json
+++ b/mdc-101/starter/package.json
@@ -11,7 +11,7 @@
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-102/complete/package.json
+++ b/mdc-102/complete/package.json
@@ -11,7 +11,7 @@
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-102/starter/package.json
+++ b/mdc-102/starter/package.json
@@ -11,7 +11,7 @@
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-103/complete/package.json
+++ b/mdc-103/complete/package.json
@@ -11,7 +11,7 @@
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-103/starter/package.json
+++ b/mdc-103/starter/package.json
@@ -11,7 +11,7 @@
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-111/complete/package.json
+++ b/mdc-111/complete/package.json
@@ -12,7 +12,7 @@
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-111/starter/package.json
+++ b/mdc-111/starter/package.json
@@ -12,7 +12,7 @@
     "babel-loader": "^7.0.0",
     "babel-preset-env": "^1.7.0",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-112/complete/package.json
+++ b/mdc-112/complete/package.json
@@ -13,7 +13,7 @@
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

--- a/mdc-112/starter/package.json
+++ b/mdc-112/starter/package.json
@@ -13,7 +13,7 @@
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
     "css-loader": "^0.28.0",
-    "dart-sass": "^1.17.2",
+    "dart-sass": "^1.25.0",
     "extract-loader": "^1.0.2",
     "fibers": "^4.0.0",
     "file-loader": "^1.1.11",

